### ISSUE TITLE
Zergpoolcoins SHIB

### ISSUE THREAD

**Issue by IwasthefirstNerrdAlerrt** [RainbowMiner/RainbowMiner#2397]
Zergpool allows to set SHIB wallet for auto exchange, but SHIB is invalid on zergpoolcoins. Can you add SHIB to zergpoolcoins as well?

**Comment by RainbowMiner** [comment]
Sure, I will add it asap.

**Comment by RainbowMiner** [comment]
Ok, now you can easily add SHIB via the console setup script.

If you want to do it by hand: add SHIB and set AECurrency in pools.config.txt:
```
"ZergPoolCoins": {
  "SHIB": "YOUR_SHIB_WALLET",'
...
  "AECurrency": "SHIB",
...
},
```

That should do the job.

**Comment by IwasthefirstNerrdAlerrt** [comment]
Appreciate it!

**Comment by IwasthefirstNerrdAlerrt** [comment]
I tried to add SHIB via (p)ool, shib and then I get invalid. Will this get pushed in an update or do I need to do something special?  Thanks for your help. I’m new to this

**Comment by RainbowMiner** [comment]
Yes, either you download the current master.zip and extract it over your current install of RainbowMiner, or you wait for the upcoming version update (within the next 48 hours).

**Comment by RainbowMiner** [comment]
SHIB on ZergPoolCoins: ZergPool was removed from RainbowMiner on 2025-09-27 (b4f6f448). Closing as obsolete for the current master (v5.0.2.1) - please open a new issue if this still occurs. Thanks for the report!
